<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_loadimg1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>690</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image Input 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_window1">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>40</y>
      <width>550</width>
      <height>550</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_loadimg2">
    <property name="geometry">
     <rect>
      <x>815</x>
      <y>690</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image Input 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_window2">
    <property name="geometry">
     <rect>
      <x>625</x>
      <y>40</y>
      <width>550</width>
      <height>550</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_saveimg">
    <property name="geometry">
     <rect>
      <x>1604</x>
      <y>692</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Image (Last Result)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clnW1">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>610</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Window 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clnW2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>610</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Window 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_window3">
    <property name="geometry">
     <rect>
      <x>1225</x>
      <y>40</y>
      <width>550</width>
      <height>550</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clnW3">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>610</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Window 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_inpu1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>11</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>WINDOW 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_input2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>11</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>WINDOW 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_output">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>11</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>OUTPUT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_grayscale">
    <property name="geometry">
     <rect>
      <x>1225</x>
      <y>692</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Grayscale Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_dtctexprssion">
    <property name="geometry">
     <rect>
      <x>1416</x>
      <y>692</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect Expression</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <action name="actionBrightAdj">
   <property name="text">
    <string>Brigthness</string>
   </property>
  </action>
  <action name="actionCtrstAdj">
   <property name="text">
    <string>Contrast</string>
   </property>
  </action>
  <action name="actionCtrst_StrcAdj">
   <property name="text">
    <string>Contrast Stretching</string>
   </property>
  </action>
  <action name="actionNgte">
   <property name="text">
    <string>Negative</string>
   </property>
  </action>
  <action name="actionNgte_G">
   <property name="text">
    <string>Negative with Grayscale</string>
   </property>
  </action>
  <action name="actionBin">
   <property name="text">
    <string>Binarization</string>
   </property>
  </action>
  <action name="actiongG_Hstgrm">
   <property name="text">
    <string>Grayscaled</string>
   </property>
  </action>
  <action name="actionRGB_Hstgrm">
   <property name="text">
    <string>RGB (Ori File)</string>
   </property>
  </action>
  <action name="actionEqualizer">
   <property name="text">
    <string>Equalizer</string>
   </property>
  </action>
  <action name="actionTranslation">
   <property name="text">
    <string>Translation</string>
   </property>
  </action>
  <action name="actionCropping">
   <property name="text">
    <string>Cropping</string>
   </property>
  </action>
  <action name="action45_Degress">
   <property name="text">
    <string>45 Degress</string>
   </property>
  </action>
  <action name="action_45_Degress">
   <property name="text">
    <string>90 Degress</string>
   </property>
  </action>
  <action name="action180_Degress">
   <property name="text">
    <string>180 Degress</string>
   </property>
  </action>
  <action name="actionmin45_Degress">
   <property name="text">
    <string>min45 Degress</string>
   </property>
  </action>
  <action name="actionmin90_Degress">
   <property name="text">
    <string>min90 Degress</string>
   </property>
  </action>
  <action name="action2x">
   <property name="text">
    <string>2x</string>
   </property>
  </action>
  <action name="action3x">
   <property name="text">
    <string>3x</string>
   </property>
  </action>
  <action name="action4x">
   <property name="text">
    <string>4x</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="text">
    <string>1/2</string>
   </property>
  </action>
  <action name="action1_4">
   <property name="text">
    <string>1/4</string>
   </property>
  </action>
  <action name="action5_4">
   <property name="text">
    <string>5/4</string>
   </property>
  </action>
  <action name="actionAdder">
   <property name="text">
    <string>Adder</string>
   </property>
  </action>
  <action name="actionSubtract">
   <property name="text">
    <string>Subtract</string>
   </property>
  </action>
  <action name="actionMultiplie">
   <property name="text">
    <string>Multiplie</string>
   </property>
  </action>
  <action name="actionDivide">
   <property name="text">
    <string>Divide</string>
   </property>
  </action>
  <action name="actionAND">
   <property name="text">
    <string>AND</string>
   </property>
  </action>
  <action name="actionOR">
   <property name="text">
    <string>OR</string>
   </property>
  </action>
  <action name="actionXOR">
   <property name="text">
    <string>XOR</string>
   </property>
  </action>
  <action name="actionRotation">
   <property name="text">
    <string>Rotation</string>
   </property>
  </action>
  <action name="actionResizing">
   <property name="text">
    <string>Resizing</string>
   </property>
  </action>
  <action name="actionSbl">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionRbts">
   <property name="text">
    <string>Robert</string>
   </property>
  </action>
  <action name="actionPrwt">
   <property name="text">
    <string>Prewitt</string>
   </property>
  </action>
  <action name="actionLplc">
   <property name="text">
    <string>Laplacian</string>
   </property>
  </action>
  <action name="actionLoG">
   <property name="text">
    <string>LoG</string>
   </property>
  </action>
  <action name="actionCanny">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
  <action name="actionFT">
   <property name="text">
    <string>Vanilla</string>
   </property>
  </action>
  <action name="actionDFT">
   <property name="text">
    <string>Discrete</string>
   </property>
  </action>
  <action name="actionGaussian">
   <property name="text">
    <string>Gaussian Filter</string>
   </property>
  </action>
  <action name="actionMedian">
   <property name="text">
    <string>Median Filter</string>
   </property>
  </action>
  <action name="actionMax">
   <property name="text">
    <string>Max Filter</string>
   </property>
  </action>
  <action name="actionConvKn1">
   <property name="text">
    <string>Kernel 1</string>
   </property>
  </action>
  <action name="actionConvKn2">
   <property name="text">
    <string>Kernel 2</string>
   </property>
  </action>
  <action name="actionSharpKn1">
   <property name="text">
    <string>Kernel 1</string>
   </property>
  </action>
  <action name="actionSharpKn2">
   <property name="text">
    <string>Kernel 2</string>
   </property>
  </action>
  <action name="actionSharpKn3">
   <property name="text">
    <string>Kernel 3</string>
   </property>
  </action>
  <action name="actionSharpKn4">
   <property name="text">
    <string>Kernel 4</string>
   </property>
  </action>
  <action name="actionSharpKn5">
   <property name="text">
    <string>Kernel 5</string>
   </property>
  </action>
  <action name="actionSharpKn6">
   <property name="text">
    <string>Kernel 6</string>
   </property>
  </action>
  <action name="actionSharpKn7">
   <property name="text">
    <string>Kernel 7</string>
   </property>
  </action>
  <action name="actionMean2x2">
   <property name="text">
    <string>2 x 2</string>
   </property>
  </action>
  <action name="actionMean3x3">
   <property name="text">
    <string>3 x 3</string>
   </property>
  </action>
  <action name="actionAdder_2">
   <property name="text">
    <string>Adder</string>
   </property>
  </action>
  <action name="actionSubs">
   <property name="text">
    <string>Substract</string>
   </property>
  </action>
  <action name="actionAnd">
   <property name="text">
    <string>AND</string>
   </property>
  </action>
  <action name="actionOr">
   <property name="text">
    <string>OR</string>
   </property>
  </action>
  <action name="actionNOT">
   <property name="text">
    <string>NOT</string>
   </property>
  </action>
  <action name="actionXor">
   <property name="text">
    <string>XOR</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Adder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
